supernode/server/router.go

```diff
@@ -20,6 +21,7 @@ func initRoute(s *Server) *mux.Router {
 	handlers := []*HandlerSpec{
 		// system
 		{Method: http.MethodGet, Path: "/_ping", HandlerFunc: s.ping},
+		{Method: http.MethodGet, Path: "/version", HandlerFunc: version.HandlerWithCtx},
```

Comment:
The response should be `VersionInfo` instead of `DragonflyVersion ` according to definition in the `swagger.yml`.

Reply:
I have unified it.